### ISSUE TITLE
New portfolio thumbnails are smaller

### ISSUE THREAD

**Issue by endycool** [wpexplorer/wpex-elegant#5]
I haven't created portfolio items for a while, and now as I'm adding new ones I notice that the thumbnails in the grid are smaller. Honestly, I don't remember if I have modified something on the previous items so that the thumbnails appear at a different size... I would like to know if this was a change on the newer versions of the theme or if I'm doing something wrong. What confuses me is that this only affects the newer items and not the previously published ones

Here's a link to my website:
http://www.innerlightleaks.com/portfolio/

Thanks!

**Comment by wpexplorer** [comment]
@endycool -By default the images should be cropping to 400x260 as you can see from the add_image_size code in functions.php - https://github.com/wpexplorer/wpex-elegant/blob/master/functions.php - can you try running a regenerate plugin to see if it fixes them? I personally use and like this one: https://wordpress.org/plugins/force-regenerate-thumbnails/

**Comment by endycool** [comment]
Thanks for the reply! I used a similar plugin to regenerate the thumbnails, and now that I know where to find that code in functions.php I might adapt it to my design.
Thanks again!

**Comment by wpexplorer** [comment]
@endycool - If you want to alter the default sizes I wouldn't recommend editing functions.php because if you ever update the theme you will lose the tweaks. You can re register the image sizes via a child theme though or use a plugin to override the defaults. Maybe I'll add some options though in the next version to do it in the Customizer ;)

**Comment by Liquidream** [comment]
I too have always found inconsistencies with the thumbnail size on the homepage.
![screen shot 2018-10-24 at 06 37 07](https://user-images.githubusercontent.com/2211678/47408962-25245780-d759-11e8-9099-1981c113791e.png)

The only way I've found to get consistent results, is to always upload a 512x512 image/gif to use as a featured image. If I mix and match image dimensions, the thumbnails are displayed at varying heights on the homepage (and I've tried regen thumbnails several times).

I'm ok for now, coz I have my workaround (always use square images, 512x512), but would be nice not to. 🤷‍♂️
